src/components/ButtonPanel.js

```diff
@@ -1,38 +1,15 @@
 import React from 'react';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div className="buttonPanel">
-    <div>
-      <Button value="AC" />
-      <Button value="+/-" />
-      <Button value="%" />
-      <Button value="&divide;" />
-    </div>
-    <div>
-      <Button value="7" />
-      <Button value="8" />
-      <Button value="9" />
-      <Button value="X" />
-    </div>
-    <div>
-      <Button value="4" />
-      <Button value="5" />
-      <Button value="6" />
-      <Button value="-" />
-    </div>
-    <div>
-      <Button value="1" />
-      <Button value="2" />
-      <Button value="3" />
-      <Button value="+" />
-    </div>
-    <div>
-      <Button value="0" />
-      <Button value="." />
-      <Button value="=" />
-    </div>
-  </div>
-);
+const ButtonPanel = () => {
+  const symbols = ['AC', '+/-', '%', '/', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '0', '='];
+  // eslint-disable-next-line react/no-array-index-key
+  const calculatorSymbols = symbols.map((x, idx) => <Button key={idx} value={x} />);
```

Comment:
You have done well doing this 👍 .

Quick one: It would be nice if you could take your time on the pathwright to read the instructions before jumping into coding, that way, it possible for your CRR to get approved in just one review.

let's look at this 👇 

![image](https://user-images.githubusercontent.com/31897434/102311289-e4a57a80-3f6c-11eb-91d4-5aae1e8bb51f.png)
https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376016/

Let try to group the buttons into 5 groups.
example below:

```
const buttonGroups = {
groupOne: {
    id: 1,
    group: ['AC', '+/-', '%', '÷'],
  },
  }
```
```
const ButtonPanel = () => (
  <>
    <div className="groupOne">
      {buttonGroups.groupOne.group.map(button => (
        <Button name={button} key={buttonGroups.groupOne.id} />
      ))}
    </div>
    )
```

**I trust you can work on the code before merging 👍**